username_api.py

```diff
@@ -118,9 +144,35 @@ def check_username(website, username):
             'url': url,
             'avatar': get_avatar(website, username),
             'possible': possible,
+            'usable': usable,
         }
 
 
+def check_usable(website):
+    """
+        Check if the website is usable.
+        Without checking the website is usable,
```

Comment:
I don't think this one is very helpful. 

i meant something like, websites go into maintenance unexpectedly. this function checks for verifies if they're working by checking for "known taken" usernames. the checking is rate limited by caching. 

For someone who plans to contribute in the future, i think such a doc string will be helpful to let the know the purpose. pls improve this

Reply:
got it.